src/connectors/siriusxm-player.ts

```diff
@@ -1,14 +1,19 @@
 export {};
 
 /*
- * On some "decade" stations SiriusXM appends the last two digits
+ * Station Meta Filter
+ * - On some "decade" stations SiriusXM appends the last two digits
  * of the release year to the song title like this: Where Is The Love (03).
+ * - On The Covers Channel, it adds Heroes (David Bowie Cover) to the track
+ * - A lot of stations seem to employ adding EXCLUSIVE to the track name, so that is now removed
  *
- * This filter removes such suffixes.
+ * Filter Station Meta
+ * - There is now a filter for much of the station meta on SiriusXM. Removes social media handles, URLs, shownames
  */
 
-const filter = MetadataFilter.createFilter({ track: removeYear });
-const removeYearRe = /\s\(\d{2}\)\s?$/g;
+const filter = MetadataFilter.createFilter({ track: removeStationMeta });
+const removeExtraRe =
+	/\s\(\d{2}\)\s?|\([^)]*\b(?:cover|Cover)\b[^)]*\)|EXCLUSIVE/gi;
```

Comment:
Removing "cover" for that case seems like a good improvement; However, I imagine there are a non-zero number of songs which legitimately end in (... cover). Misinterpreting those is not a big deal, but if there's any way to access the channel information, we could cut down that false negative (positive?) rate.

Reply:
@jaccarmac I think pulling in the show name wouldn't be too hard. I updated the Global Player connector recently and they had a show name filter, so the idea of filtering based on the channel meta seems reasonable.

Reply:
@jaccarmac I separated the 3 regex criteria into different filters, though didn't do the channel identification. I did a search on [XMPlaylist](https://xmplaylist.com) and see there are a bunch of channels that do have cover labels, and we'd really have no idea which ones might have "cover)" as part of the official title of the song. I think it's an all or nothing proposition. I searched how some of them are labelled in Apple Music, and it's without the mention of being a cover. Though you can see many are exclusive, so may not exist anywhere aside from on SXM. Which I suppose doesn't matter, since last.fm is just logging the listen, not the validity or findability of the song.  

![image](https://github.com/web-scrobbler/web-scrobbler/assets/2962327/cdc0842c-fbbf-471b-92db-1f904a9b0ed6)